src/main/resources/data/galacticraft/recipes/crafting_table/decoration/dark/dark_decoration_stairs.json

```diff
@@ -0,0 +1,17 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern": [
+    "L  ",
+    "LL ",

```

Comment:
There has to be another recipe with the stair facing the opposite direction in the crafting table.